### PR TITLE
Update Mattsi Jansky profile with up-to-date info and more flattering image

### PR DESCRIPTION
**Pull request by Mattsi-Jansky**

### CODE REVIEW

**Comment by josewenzel** [comment]
It looks good 👍 nice picture @Mattsi-Jansky

**Comment by ghost** [comment]
I prefer the old picture, but we would miss your gainz. Approved :)

**Review by ghost — APPROVED** [review]

**Review by josewenzel — APPROVED** [review]

File: src/_data/team.yml [hunk 1]
```diff
@@ -308,12 +308,14 @@
   image: /assets/custom/img/team/mattsi_jansky.jpg
   role: Software Craftsman
   fullDescription:
-    <p>Mattsi is an ambitious software developer that loves to learn and belives in "doing it right the first time", prioritising quality software over risky shortcuts.</p>
-    <p>Before joining Codurance he experienced a broad career across full-stack and back-end development roles in education, international e-commerce and financial technology focusing on the ASP.Net stack.</p>
+    <p>Mattsi is an ambitious software developer that loves to learn and belives in "doing it right the first time", putting quality software first.</p>
```
Comment (apavlidi) [inline src/_data/team.yml:311]: @Mattsi-Jansky Just found a typo in the `belives` word :blush:

Comment (Mattsi-Jansky) [reply src/_data/team.yml:311]: Oh man, I can't believe no-one's noticed that until now 😆 Thanks!

Comment (ghost) [reply src/_data/team.yml:311]: argh :(

**Pull request merged**